### PR TITLE
Fix UnzipFile Script

### PR DESCRIPTION
**Pull request by Shellyber**
## Status
Ready

## Related Issues
fixes: https://github.com/demisto/etc/issues/20557

## Description
Fixed an issue where zip file returned with 0 file size

## Does it break backward compatibility?
   - No

## Must have
- [ ] Tests
- [ ] Documentation (with link to it)
- [ ] Code Review

## Dependencies
Mention the dependencies of the entity you changed as given from the precommit hooks in checkboxes, and tick after tested them.
- [ ] Dependency 1
- [ ] Dependency 2
- [ ] Dependency 3

### CODE REVIEW

**Comment by Shellyber** [comment]
@kirbles19 Please review the CHANGELOG file.

File: Scripts/UnzipFile/UnzipFile.py [hunk 1]
```diff
@@ -6,9 +6,14 @@
 from os.path import isdir
 from os.path import isfile
 from subprocess import Popen, PIPE
+from tempfile import mkdtemp
+import shutil
 
 
-def main():
+dir_path = mkdtemp()
```
Comment (yaakovi) [inline Scripts/UnzipFile/UnzipFile.py]: move to the bottom (try block under the `if __name__ ...`)

**Comment by Shellyber** [comment]
Maybe better to use `shlex` anyway?

**Comment by kirbles19** [comment]
@Shellyber Done.

**Pull request merged**